src/components/common/DetailsList.vue

```diff
@@ -34,36 +35,36 @@
 </template>
 
 <script lang="ts">
-// Libraries
 import { Component, Mixins, Prop, Emit } from 'vue-property-decorator'
-import { mapGetters } from 'vuex'
-
-// Mixins
+import { Getter } from 'vuex-class'
 import { DateMixin } from '@/mixins'
+import { HistoryItemIF, TodoItemIF } from '@/interfaces'
 
-@Component({
-  computed: {
-    ...mapGetters(['isRoleStaff'])
-  }
-})
+@Component({})
 export default class DetailsList extends Mixins(DateMixin) {
-  /** Filing containing correction information. */
-  @Prop({ default: () => { return { comments: [] } } })
-  private filing: object
+  /** The filing containing comments. */
+  @Prop({ default: () =>
+    ({
+      comments: [],
+      filingId: null
+    })
+  })
```

Comment:
@cameron-freshworks Here's some new syntax for your know-how. (It was new to me!)

-> for arrow functions, instead of `() => return { xxx }`, just use `() => ({ xxx })`